### PR TITLE
Partially revert #18070 (Add support for Windows CA certificate store)

### PR DESCRIPTION
**Pull request by hlandau**
As discussed at OTC. Blocker for 3.2. I kept and tweaked the documentation and the store itself.

### CODE REVIEW

File: doc/man3/X509_get_default_cert_file.pod [hunk 1]
```diff
@@ -37,48 +32,31 @@ specified. If a given directory in the list exists, OpenSSL attempts to lookup
...
+recommended to specify a nondefault value to be used instead of the value
+returned by X509_get_default_cert_dir(). The value specified by this environment
+variable can also be a store URI (but see BUGS below).
+
+=head1 BUGS
+
+By default (for example, when L<X509_STORE_set_default_paths(3)> is used), the
+environment variable name returned by X509_get_default_cert_dir_env() is
+interpreted both as a delimiter-separated list of paths, and as a store URI by
+default. This is ambiguous. For example, specifying a value of
```
Comment (t8m) [inline doc/man3/X509_get_default_cert_file.pod:53]: There is duplicate `by default` in this sentence. Looks awkward to me.

File: CHANGES.md [hunk 2]
```diff
@@ -229,24 +229,13 @@ OpenSSL 3.2
...
-
  * Support for loading root certificates from the Windows certificate store
    has been added. The support is in the form of a store which recognises the
-   URI string of `org.openssl.winstore://`. This store is enabled by default and
-   can be disabled using the new compile-time option `no-winstore`.
+   URI string of `org.openssl.winstore://`. This URI scheme currently takes no
+   arguments. This store is built by default and can be disabled using the new
+   compile-time option `no-winstore`. This store is not currently used by
+   default and must be loaded explicitly using the above store URI. It is
+   expected to be loaded by default in the next minor release of OpenSSL.
```
Comment (t8m) [inline CHANGES.md:238]: Perhaps better to say - expected to be loaded by default in future.

File: doc/man3/X509_get_default_cert_file.pod [hunk 3]
```diff
@@ -96,14 +74,9 @@ L<SSL_CTX_load_verify_dir(3)>,
...
 
-=head1 HISTORY
-
-X509_get_default_cert_uri(), X509_get_default_cert_path_env() and
-X509_get_default_cert_uri_env() were introduced in OpenSSL 3.2.
-
 =head1 COPYRIGHT
 
-Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
```
Comment (t8m) [inline doc/man3/X509_get_default_cert_file.pod:79]: 2022-2023?

**Review by t8m — CHANGES_REQUESTED** [review]

**Comment by hlandau** [comment]
Updated.

**Review by t8m — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Review by levitte — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]

**Comment by paulidale** [comment]
Merged.

**Pull request closed without merging**